test/Microsoft.AspNet.Mvc.Core.Test/SessionStateTempDataProviderTest.cs

```diff
@@ -206,7 +207,7 @@ public void SaveAndLoad_SimpleTypesCanBeStoredAndLoaded()
             var boolVal = Assert.IsType<bool>(TempData["bool"]);
             Assert.Equal(false, boolVal);
             var datetimeVal = Assert.IsType<DateTime>(TempData["DateTime"]);
-            Assert.Equal(new DateTime().ToString(), datetimeVal.ToString());
+            Assert.Equal(inputDatetimeVal.ToString(), datetimeVal.ToString());
```

Comment:
Earlier `inputDateTimeVal` `DateKind` was `Unspecified` while `datetimeVal`'s `DateKind` was `Local`. With this change both `DateKind` will be `Local`. So that should fix the bug.